README.md

```diff
@@ -14,6 +15,13 @@ The virtual environment can be created locally by using the `poetry install` com
 
 New package dependencies for the project can be added using the `poetry add <package-name-from-pip>` command.
 
+### Nginx for Localhost
+To set up nginx configuration, a self signed certificate is needed. To set this up, in the terminal run:
+`openssl req -x509 -newkey rsa:4096 -keyout private.key -out cert.pem -days 365 -sha256 -nodes`
+
+Only the "Country Name" field is necessary, in which you should should type "us" then press enter. All other fields may be ignored by entering "."
```

Comment:
I believe you are supposed the just press enter here, not ".". 

Alex, is this correct?

I would just tell the user to press <kbd>Enter</kbd> for all the other fields.

Reply:
Default varies by client, some will default to blank others will have placeholder data. Using <kdb>.</kbd> will leave it blank even the default isn't